### PR TITLE
add quick-start.md ukrainian language

### PR DESCRIPTION
**Pull request by vadikcorp**
added quick-start.md in ukrainian

### CODE REVIEW

File: docs-translations/uk-UA/tutorial/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+Головний процес керує всіма веб-сторінками і відповідними їм процесами-рендерами.
+Кожний процес-рендерер ізольований і керує тільки своєю веб-сторінкою.
+
+На веб-сторінках не дозволяється викликати нативні API, які керують GUI,
+тому що це дуже небезпечно і може легко викликати витік ресурсів. Якщо ви хочете
+виконати дії з GUI на сторінці, процес-рендерер цієї сторінки повинен
+"Попросити" головний процес зробити ці дії.
+
+У Electron є декілька способів спілкування між процесам. Наприклад, модулі
+[`ipcRenderer`](../api/ipc-renderer.md) и [`ipcMain`](../api/ipc-main.md) використовується
```
Comment (TiagoDanin) [inline docs-translations/uk-UA/tutorial/quick-start.md]: "404 Page not found"
Invalid address

File: docs-translations/uk-UA/tutorial/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+```bash
+# Клонуємо репозиторій
+$ git clone https://github.com/electron/electron-quick-start
+# Переходимо в папку скачаного репозиторія
+$ cd electron-quick-start
+# Встановлюємо залежності і запускаємо
+$ npm install && npm start
+```
+
+[share-data]: ../faq/electron-faq.md#how-to-share-data-between-web-pages
```
Comment (TiagoDanin) [inline docs-translations/uk-UA/tutorial/quick-start.md]: ^ Same here

**Comment by vadikcorp** [comment]
@TiagoDanin  hey sir, i replaced the link

File: docs-translations/uk-UA/tutorial/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+Головний процес керує всіма веб-сторінками і відповідними їм процесами-рендерами.
+Кожний процес-рендерер ізольований і керує тільки своєю веб-сторінкою.
+
+На веб-сторінках не дозволяється викликати нативні API, які керують GUI,
+тому що це дуже небезпечно і може легко викликати витік ресурсів. Якщо ви хочете
+виконати дії з GUI на сторінці, процес-рендерер цієї сторінки повинен
+"Попросити" головний процес зробити ці дії.
+
+У Electron є декілька способів спілкування між процесам. Наприклад, модулі
+[`ipcRenderer`](https://electron.atom.io/docs/api/ipc-renderer/) и [`ipcMain`](https://electron.atom.io/docs/api/ipc-main/) використовується
```
Comment (TiagoDanin) [inline docs-translations/uk-UA/tutorial/quick-start.md]: Replace to `../../../docs/api/ipc-renderer.md` and `../../../docs/api/ipc-main.md`

**Comment by sethlu** [comment]
Hi @vadikcorp! We've recently [moved](https://github.com/electron/electron/blob/master/docs-translations/README.md)  our translations to [Crowdin](https://crowdin.com/projects/electron) to make it easier for contributors to make changes. Please feel encouraged to make the correction there.

I'm going to close this pull request, though, because as a result of the change there is no longer the `docs-translations/zh-TW` directory.

cc: @TiagoDanin

**Pull request closed without merging**